Makefile

```diff
@@ -90,6 +93,10 @@ ifneq ($(wildcard /.dockerenv), )
 	go test -check.v -check.timeout=$(TEST_TIMEOUT) timeout=$(TEST_SUITE_TIMEOUT) $(TESTFLAGS) github.com/docker/containerd/integration-test
```

Comment:
Can we update the `test` rule to also run the bench mark so that it's done for every PR?